packages/@aws-cdk/aws-lambda/test/function.test.ts

```diff
@@ -2172,6 +2172,30 @@ describe('function', () => {
     })).toThrow(/Layers are not supported for container image functions/);
   });
 
+  test('error when function name is invalid', () => {
+    // GIVEN
+    const stack = new cdk.Stack();
+    const tooLongName = 'M'.repeat(141);
+
+    // THEN
+    expect (() => new lambda.Function(stack, tooLongName, {
+      code: new lambda.InlineCode('foo'),
+      handler: 'index.handler',
+      runtime: lambda.Runtime.NODEJS_10_X,
```

Comment:
Why use a deprecated runtime?

Reply:
My mistake. I was using the tests near the top of the file as a reference. I'll fix.